Components/Alerts/alerts.css

```diff
@@ -1,5 +1,5 @@
 .alert{
-    width:300px;
+    width:30rem;
     text-align: center;
```

Comment:
You could use max-width here than your alert would become responsive. But that's just a suggestio this also looks fine.